### ISSUE TITLE
build error - missing config file compatible with lazperf.cmake

### ISSUE THREAD

**Issue by islapesca** [PDAL/PDAL#3641]
Cannot build PDAL onto local machine due to this error. I am attempting to add some functionality to some PDAL tools however I cannot complete the build to start this process.
Working on Windows machine, VS 2019, PDAL version 2.3.0

**Error:**
`C:\UderDirectory\Conda Test\pdal mod4 (new pull)\PDAL>cmake --build .`

CMake Error at cmake/lazperf.cmake:3 (find_package):
  Could not find a configuration file for package "LAZPERF" that is
  compatible with requested version "3.0".

  The following configuration files were considered but not accepted:

    C:/Anaconda3/Library/lib/cmake/LAZPERF/lazperf-config.cmake, version: 3.0.0 (64bit)

Call Stack (most recent call first):
  CMakeLists.txt:129 (include)

-- Configuring incomplete, errors occurred!
See also "C:\UderDirectory\Conda Test/pdal mod4 (new pull)/PDAL/CMakeFiles/CMakeOutput.log".
See also "C:\UderDirectory\Conda Test/pdal mod4 (new pull)/PDAL/CMakeFiles/CMakeError.log".
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe"' : return code '0x1'
Stop.`

**Comment by abellgithub** [comment]
It's hard to know what's wrong given the information you've provided. We build on Windows with lazperf every day. You should look [here](https://github.com/PDAL/PDAL/tree/master/scripts/ci/win) for the process we use.

**Comment by islapesca** [comment]
Hi Andrew!

So after following the instructions, this is the resulting message:

Interestingly enough, I know that lazperf is optional, however if I go into
the cmake files and play with removing lazperf from being called, the
message I receive is that all tools have been disabled.

Any insight would be appreciated.

Best,

Anthony

**Comment by abellgithub** [comment]
You're not providing enough information. If you're building master (you said 2.3), lazperf is not optional, but again, I have no idea what you're doing. Did you try to use the configuration information that I linked above?

**Comment by islapesca** [comment]
Hello!

Thanks for getting back to me. Clearly I’m not sure what information to
provide, sorry about that. Hopefully I’m not troubling you.

Yes I tried the link you sent. I was able to run setup.sh in a Git Bash,
however compile.sh and cmake.sh do not run. I get a warning that says “file
not found" for the compile file and “cmake not a command “ for the cmake
file.


Basically, I pulled PDAL into my own GitHub repo, then cloned that repo
onto my computer. I am trying to compile the application, so that I can
test adding another functionality to the e57 tool.

Does that all make sense?

Thank you,
Anthony
